extensions/adobe/experience/offer-management/eligibility-rule.schema.json

```diff
@@ -27,9 +27,10 @@
           "description": "EligibilityRule name that is used to locate this object in a search operation"
         },
         "xdm:value": {
-          "type": "string",
-          "title": "PQL Expression",
-          "description": "The string serialization of the predicate expression in pql/json format"
+          "type": "object",
```

Comment:
you don't need `type` when you use `$ref`. Everything is ignored by validator when `$ref` is present.

Reply:
Yeah, I know. Having the type and description there helps with understanding the `xdm:value` property.